README.md

```diff
@@ -57,6 +57,7 @@ we move them into the [Alumni section](#alumni-) below.
 | Explainer | Issues | Feedback | Group |
 | --------- | ------ | -------- | -------- |
 | [CSS Gap Decorations](CSSGapDecorations/explainer.md) | <a href="https://github.com/MicrosoftEdge/MSEdgeExplainers/labels/CSSGapDecorations">![GitHub issues by-label](https://img.shields.io/github/issues/MicrosoftEdge/MSEdgeExplainers/CSSGapDecorations?label=issues)</a> | [New issue...](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/new?assignees=kbabbitt&labels=CSSGapDecorations&template=css-gap-decorations.md&title=%5BCSS+Gap+Decorations%5D+%3CTITLE+HERE%3E) | CSS |
+| [Enterprise Platform Authentication Broker API](PlatformAuthentication/explainer.md) | <a href="https://github.com/MicrosoftEdge/MSEdgeExplainers/labels/PlatformAuthentication">![GitHub issues by-label](https://img.shields.io/github/issues/MicrosoftEdge/MSEdgeExplainers/PlatformAuthentication?label=issues)</a> | [New issue...](https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/new?assignees=coder-linping&labels=PlatformAuthentication&template=platform-authentication-api.md&title=%5BPlatform+Authentication+API%5D+%3CTITLE+HERE%3E) | JavaScript |
```

Comment:
This is not a JavaScript feature (it doesn't affect the language or something closely associated with it). Let's instead label it `Identity`.